tslint-warnings.csv

```diff
@@ -156,6 +156,7 @@ no-redundant-jsdoc,Forbids JSDoc which duplicates TypeScript functionality.,TSLI
 no-reference-import,Don't `<reference types="foo" />` if you import `foo` anyway.,TSLINT178RR3K,tslint,Non-SDL,Warning,Important,Opportunity for Excellence,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,710,"CWE 710 - Coding Standards Violation"
 no-regex-spaces,Do not use multiple spaces in a regular expression literal. Similar to the ESLint no-regex-spaces rule,TSLINT1T5TJ80,tslint,Non-SDL,Error,Critical,Opportunity for Excellence,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,,
 no-reserved-keywords,"Do not use reserved keywords as names of local variables, fields, functions, or other identifiers.",TSLINT14J77I2,tslint,SDL,Error,Critical,Mandatory,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,398,"CWE 398 - Indicator of Poor Code Quality"
+no-restricted-globals,Disallow specific global variables.,TSLINTGKU2KB,tslint,,,,,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,,
```

Comment:
Unexpected change, if correct version of packages is installed for this branch.

Reply:
pulled latest master version + run npm install,
still this entry is there. 😮

Reply:
Thanks for pinging. Will check what's with this line today evening.

Reply:
You are correct. Looks like these strings were not added during update of `tslint` package.